### ISSUE TITLE
Docker Desktop Edge does not use gRPC-FUSE by default (contradicting the release notes)

### ISSUE THREAD

**Issue by PerMildner** [docker/for-mac#4864]
- [ x] I have tried with the latest version of my channel (Stable or Edge)
  - [ x] I have uploaded Diagnostics
  - Diagnostics ID: AE1EFB8B-27FD-4A63-925F-369B4BF5C8D5/20200826093548

### Expected behavior
"Docker Desktop now uses gRPC-FUSE" as [the release notes](https://docs.docker.com/docker-for-mac/edge-release-notes/#docker-desktop-community-2350) claim.
### Actual behavior
The preferences has "Use gRPC-FUSE for file sharing" unchecked both after "Reset to factory defaults" and after uninstalling and reinstalling.

### Information

  - macOS Version:
10.15.6 (19G2021)
### Diagnostic logs

```
Docker for Mac: 2.3.5.0 (47376) edge (would be nice if it was possible to copy this from the About box...)

```

**Comment by stephen-turner** [comment]
It is actually used, but the checkbox doesn't correctly reflect the state. ☹️ It will be fixed in the next release.

**Comment by djs55** [comment]
The fix for this issue has been released in Edge 2.3.6.1. Thanks again for the report.
